include_physics/ideal_MHD.h

```diff
@@ -387,14 +387,300 @@ inline void init_fields(uint ix, uint iy, uint iz, global REAL* u) {
 // Surface terms
 //--------------------------------------------------------------------------------------------------
 
+#ifndef USE_PERIODIC
+
+#define sq(x) ((x)*(x))
+#define PLUS(x) (max(x, 0.0))
+
+inline void calc_flux_f(REAL *u, REAL *flux) {
```

Comment:
In order to be consistent with the notation used in the upper parts, this should be `calc_flux_x`. But we can also leave it as it is, if you like.
Adding some `const` to `u` might be nice to indicate input and output variables.